.github/workflows/pr_check_workflow.yml

```diff
@@ -189,7 +190,7 @@ jobs:
       # github virtual env is the latest chrome
       - name: Setup chromedriver
         if: steps.ftr_tests_results.outputs.ftr_tests_results != 'success'
-        run: yarn add --dev chromedriver@97.0.0
```

Comment:
Got example of this running? Now reflecting I thinking this should generally be fine to keep the same since the env was requiring chromedriver@97

Reply:
Well we run into issues either way right? Either we have to update it every time GitHub updates it, or it fails until GitHub updates to the latest...

Reply:
Yeah it's a pain! I wonder if we can create an issue to refactor this to use the appropriate image https://hub.docker.com/r/opensearchstaging/ci-runner/tags instead of relying purely on github virtual env.

Do we know how often chromedriver gets updated versus github virtual env consuming it?

Reply:
It'll be updated to 98 later this week: https://github.com/actions/virtual-environments/issues/5045

Reply:
I reverted this change.

Reply:
For sure. Yeah I'll create an issue just to run all these tests in a docker container pulled from opensearchstaging.